website/docs/faqs/Accounts/cloud-upgrade-instructions.md

```diff
@@ -32,17 +32,20 @@ To unlock your account and select a plan, review the following guidance per plan
 3. Confirm your plan selection on the pop up message. 
 4. This automatically unlocks your dbt Cloud account, and you can now enjoy the benefits of the Developer plan. 🎉
 
```

Comment:
i think we mentioned that you can add the bit here about the overage pop up if a user has more than 1 user/project and is downgrading to developer?